Makefile

```diff
@@ -240,6 +244,11 @@ docopen: out/doc/api/all.html
 docclean:
 	-rm -rf out/doc
 
+run-ci:
+	$(PYTHON) ./configure --without-snapshot $(CONFIG_FLAGS)
```

Comment:
Just an observation and a potential follow-up to this PR: if the goal of `run-ci` is to test builds of node with the same configuration as official releases, we should maybe add other configuration options used for them (such as `--with-intl=small-icu`). We could probably factor that out in a variable and reuse that in other targets used to build official releases (like `binary` and `pkg`).

Same for the `test-ci` command line option supported in `vcbuild.bat` below.